apps/prairielearn/src/server.js

```diff
@@ -1298,6 +1298,26 @@ export async function initExpress() {
     },
     (await import('./pages/instructorQuestions/instructorQuestions.js')).default,
   ]);
+  app.use('/pl/course_instance/:course_instance_id(\\d+)/instructor/ai_generate_question_audit', [
```

Comment:
Let's be consistent with naming here, you use "audit" and "job review" to refer to the same page. IMO we should call this page `ai_generate_question_jobs`, which will trickle down into file/function/component/etc. naming.

The page to view a single job can then be called `ai_generate_question_job`.

Reply:
Changed in 9ec48833185d75e4ebfa4e4bb5a1682f21abe892.